### DISCUSSION TITLE
problem loading mesh from msh file

### DISCUSSION THREAD

**Discussion by RWTHLHK** [idaholab/moose#29008] [Q&A Meshing] [answered]
I generated a msh file using Neper. But weirdly, the boundary name in the msh file can never be identified. I always get the error: the following node sets (ids) do not exist on the mesh: y0 (105). But I check in the msh file, boundary y0 indeed exists.
[BCs]
  [symmy]
    type = DirichletBC
    variable = uy
    boundary = y0
    value = 0
  []
  [symmx]
    type = DirichletBC
    variable = ux
    boundary = x0
    value = 0
  []
  [symmz]
    type = DirichletBC
    variable = uz
    boundary = z0
    value = 0
  []
  [tdisp]
    type = FunctionDirichletBC
    variable = uz
    boundary = z1
    function = 0.01*t
  []
[]
![image](https://github.com/user-attachments/assets/f24f5aef-8e35-4246-98a6-e0910498f1a2)

**Comment by GiudGiud** [comment, marked as answer]
hello

is it a sideset in this msh file instead of a nodeset?
if so you can use this to convert it:
https://mooseframework.inl.gov/source/meshgenerators/NodeSetsFromSideSetsGenerator.html

**Reply by RWTHLHK** [reply]
thanks! I just found BoundingBoxNodeSetGenerator can also do the job.
